roles/edpm_reboot/defaults/main.yaml

```diff
@@ -17,7 +17,8 @@
 
 # All variables intended for modification should be placed in this file.
 
-edpm_reboot_force_reboot: false
+edpm_reboot_strategy: auto
+edpm_reboot_networker_node_config_dir: /var/lib/config-data/ovn_controller
```

Comment:
IIUC this should match also on legacy compute nodes, which is nice, since the nova.conf approach probably will not match those (as the config path is different in 17.1).